src/amo/components/LanguagePicker.js

```diff
@@ -47,7 +47,7 @@ export class LanguagePickerBase extends React.Component {
     return (
       <div className="LanguagePicker">
         <h3 className="LanguagePicker-header">
-          {i18n.gettext('Browse in your language')}
+          {i18n.gettext('Language')}
```

Comment:
I'm gonna go off-script and ask this be "Change Language". "Change" could even be hidden but I just feel this lacks a bit of context...

Reply:
Yeah, I like more context to this too, the only downside is breaking consistency with other properties.

Reply:
At the very least just make it a hidden `<span>` for now... maybe we can suggest an upstream patch for it though.

Reply:
I'm just going to make it visible. 👍